contracts/strategies/AaveStrategyAvaxV1.sol

```diff
@@ -165,53 +161,51 @@ contract AaveStrategyAvaxV1 is YakStrategyV2Payable, ReentrancyGuard {
         revert();
     }
 
-    function _deposit(address account, uint256 amount)
-        private
-        onlyAllowedDeposits
-    {
+    function _deposit(address account, uint256 amount) private onlyAllowedDeposits {
         require(DEPOSITS_ENABLED == true, "AaveStrategyV1::_deposit");
         if (MAX_TOKENS_TO_DEPOSIT_WITHOUT_REINVEST > 0) {
             uint256 avaxRewards = _checkRewards();
             if (avaxRewards > MAX_TOKENS_TO_DEPOSIT_WITHOUT_REINVEST) {
                 _reinvest(avaxRewards);
             }
         }
-        uint256 WAVAXAmount = amount;
-        uint256 balance = _totalDepositsFresh();
-        // if (totalSupply.mul(balance) > 0) {
-        //     WAVAXAmount = amount.mul(totalSupply).div(balance);
-        // }
-        _mint(account, WAVAXAmount);
+        _mint(account, amount);
         _stakeDepositTokens(amount);
         emit Deposit(account, amount);
     }
 
     function withdraw(uint256 amount) external override nonReentrant {
-        require(amount > minMinting, "AaveStrategyV1:: below minimum withdraw");
-        uint256 WAVAXAmount = _totalDepositsFresh().mul(amount).div(
-            totalSupply
-        );
+        uint256 WAVAXAmount = _totalDepositsFresh().mul(amount).div(totalSupply);
+        require(WAVAXAmount > minMinting, "AaveStrategyV1::below minimum withdraw");
```

Comment:
A find and replace all to fix the namespace :)

`AaveStrategyV1::below minimum withdraw`